src/openms/include/OpenMS/ANALYSIS/OPENSWATH/SpectrumExtractor.h

```diff
@@ -52,12 +52,20 @@
 namespace OpenMS
 {
   /**
-  @brief TODO BRIEF EXPLANATION HERE
-
-  TODO MULTILINE
-  EXPLANATION
-  HERE
-
+    @brief This class implements algorithms that, given an experiment and a target
```

Comment:
"This class filters, annotates, picks, and scores spectra (e.g., taken from a DDA experiment) based on a target list.